bot/cogs/help.py

```diff
@@ -0,0 +1,239 @@
+"""Help system.
+
+This help is nearly perfect, with only one warning (access to a protected
+class) and two info (not enough methods at line 16 and too many arguments at
+line 19), under Flake8, Mypy, Pydocstyle and Pylint.
+"""
+
+from inspect import Parameter
+import textwrap
+import typing as t
+
+import discord
+from discord.ext import commands, menus
+import discord.utils
+
+
+class HelpSource(menus.ListPageSource):
+    """The Help menu."""
+
+    def __init__(
+            self,
+            signature: t.Callable,
+            filter_commands: t.Coroutine,
+            prefix: str,
+            author: discord.User,
+            cogs: dict
+            ) -> None:
+        """Create the menu."""
+        self.get_command_signature = signature
+        self.filter_commands = filter_commands
+        self.prefix = prefix
+        self.menu_author = author
+        super().__init__(
+            [(cog, cogs[cog]) for cog in sorted(
+                cogs,
+                key=lambda cog: cog.qualified_name if cog else "ZZ"
+            )],
+            per_page=1,
+        )
+
+    async def format_page(
+            self,
+            menu: menus.Menu,
+            cog_tuple: tuple) -> discord.Embed:
```

Comment:
Apply the new proposed function parameters syntax as described above to this function **and all other function which doesn't respect it**
```suggestion
    async def format_page(
        self,
        menu: menus.Menu,
        cog_tuple: tuple
    ) -> discord.Embed:
```

Reply:
Okay